src/modeler/AutoRest.Swagger/Validation/Core/RulesExtensions.cs

```diff
@@ -64,6 +64,6 @@ public static IEnumerable<Rule> GetValidationRules(this Type type)
         /// <summary>
         /// The collection of rules that apply to all properties regardless of other rules.
         /// </summary>
-        public static IEnumerable<Rule> UniversalRules = new[] { new NoControlCharacters() };
+        public static IEnumerable<Rule> UniversalRules = new List<Rule>();
```

Comment:
A. NoControlCharacters was the only universal rule implemented from what I can tell.
B. "We shouldn't ever add a universal rule." Not sure about that, rather I don't trust the ever changing requirements for validator.
C. I agree it would be cleaner on the ts side of things, but until that is enabled let's humor this functionality, what say?